### PR TITLE
Code/js/main

### PR DESCRIPTION
**Pull request by ntohidi**
I finished my main `JS` tasks plz check and verify and add to my `kportal` repo 🤓

- [x] used function
- [ ] used list comprehension

### CODE REVIEW

**Comment by unclecode** [comment]
well done, I will check and update you

File: task1/task.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
 #Write a function to print even numbers from 1 to 10
 def printEven():
+    for x in [1..10]:
```
Comment (unclecode) [inline task1/task.py]: you should use `range` function, change it.

Comment (unclecode) [inline task1/task.py]: change it to `range()`

File: task2/task.py [hunk 2]
```diff
@@ -1,3 +1,4 @@
 #Create a function get a number and returns true if its even anf alse if its odd
 def checkMe(n):
+    return n % 2 == 0
```
Comment (unclecode) [inline task2/task.py]: use `if` instead

**Review by unclecode — CHANGES_REQUESTED** [review]
Check task 1 and task 2 for changes

File: task1/task.py (hunk 1 above)
Comment (ntohidi) [reply task1/task.py]: forgot how was `range`

Comment (unclecode) [reply task1/task.py]: very simple, like this `range(1,10)`

**Review by ntohidi — COMMENTED** [review]
Plz check it I guess it's ready

**Review by tomkidoz — COMMENTED** [review]
I am checking

File: task1/task.py [hunk 3]
```diff
@@ -1,3 +1,6 @@
 #Write a function to print even numbers from 1 to 10
 def printEven():
+    for x in range(1,10):
```
Comment (tomkidoz) [inline task1/task.py]: you should include the `10` itself now its till `10`

**Review by tomkidoz — CHANGES_REQUESTED** [review]
Okay i have one comment on task1

File: task1/task.py (hunk 3 above)
Comment (unclecode) [reply task1/task.py]: Nice comment @tomkidoz , @ntohidi do this

Comment (ntohidi) [reply task1/task.py]: Done

**Pull request merged**